toolbox/incari/object/get-position.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+
+![The Get Position Node.](../../../.gitbook/assets/toolbox/incari/object/get-position.PNG)
+
+| Input | Output |
+| :--- | :--- |
+| Object ID | Position |
+
+**Get Position** returns the *position* of an **Object** in a **Scene**. The **Object ID** is given as input to the **Node**.
+
+# Inputs
+
+|Input|Type|Description|
+|---|---|---|
+|*Pulse Input* (►)|**Pulse**|A standard **Input Pulse**, to trigger the execution of the **Node**.|
+|`Object ID`|Int|The ID of the **Object** whose *position* is to be determined.
+
+# Outputs
+
+|Output|Type|Description|
+|---|---|---|
+|*Pulse Output* (►)|**Pulse**|A standard **Output Pulse**, to move onto the next **Node** along the **Logic Branch**, once this **Node** has finished its execution.|
+|`Position`| 3D vector | A 3 dimensional vector that contains x, y and z positions.
```

Comment:
The data type should be Vector3. If you use the `io` snippet and press tab to move through to different points, you should get a drop-down list of data types which has Vector3 listed. Alternatively, you can use the `data-type` prefix, which also gives a dropdown. If the snippets aren't showing up, press Ctrl + Space to see them.